Snippets/ASBFunctions/ASBFunctions_1.3/FunctionsHostBuilderUsage.cs

```diff
@@ -0,0 +1,141 @@
+using NServiceBus;
+
+// needs to be top level first defined for C# to compile
+#region endpoint-trigger-function-wireup
+
+[assembly: NServiceBusTriggerFunction("MyFunctionsEndpoint",
+    TriggerFunctionName = "NServiceBusFunctionEndpointTrigger-ASBTriggerQueue")]
```

Comment:
any reason that we specifically define the function name?

Reply:
no reason, just copied code from the earlier version. I'll remove.